Azure/sonic-buildimage
# [DellEMC Z9332f] Remove duplicate ipmihelper.py script

Signed-off-by: Danny Allen <daall@microsoft.com>



**- Why I did it**
Fixes https://github.com/Azure/sonic-buildimage/issues/6445

Because the ipmihelper.py script in the 9332 folder is slightly different than the common one (due to LGTM fixes), when the common one gets copied during build time it causes the workspace/build to become dirty.

**- How I did it**
I removed the ipmihelper that was added in the 9332 subdirectory and ported over the LGTM fixes that were added there.

**- How to verify it**
Build a BRCM image and verify that the version string isn't dirty.

**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
![image](https://user-images.githubusercontent.com/52468448/105554289-e1b04f00-5cbb-11eb-87fc-f2acbd22f694.png)
